docs/speech-usage.rst

```diff
@@ -151,5 +151,84 @@ words to the vocabulary of the recognizer.
     transcript: Hello, this is a test
     confidence: 0.81
 
+
+Streaming Recognition
+---------------------
+
+The :meth:`~google.cloud.speech.Client.streaming_recognize` method converts
+speech data to possible text alternatives on the fly.
+
+.. note::
+    Streaming recognition requests are limited to 1 minute of audio.
+
+    See: https://cloud.google.com/speech/limits#content
+
+.. code-block:: python
+
+    >>> from google.cloud import speech
+    >>> client = speech.Client()
+    >>> with open('./hello.wav', 'rb') as stream:
+    ...     sample = client.sample(content=stream,
+    ...                            encoding=speech.Encoding.LINEAR16,
+    ...                            sample_rate=16000)
+    ...     response = list(client.streaming_recognize(sample))
+    ...     print(response[0].transcript)
+    'hello'
+    ...     print(response[0].confidence)
+    0.973458576
+
+
+By default the recognizer will perform continuous recognition
+(continuing to process audio even if the user pauses speaking) until the client
+closes the output stream or when the maximum time limit has been reached.
+
+If you only want to recognize a single utterance you can set
+ ``single_utterance`` to ``True`` and only one result will be returned.
+
+See: `Single Utterance`_
+
+.. code-block:: python
+
+    >>> with open('./hello_pause_goodbye.wav', 'rb') as stream:
+    >>>     sample = client.sample(content=stream,
+    ...                            encoding=speech.Encoding.LINEAR16,
+    ...                            sample_rate=16000)
+    >>>     response = client.streaming_recognize(sample,
+    ...                                           single_utterance=True)
+    >>> results = list(response)
+    >>> print(results[0].transcript)
+    hello
+    >>> print(results[0].confidence)
+    0.96523453546
```

Comment:
Although I haven't seen it, it could be 2 if `max_alternatives` is set to 2. In practice I've just seen one.